core/src/main/java/greencity/security/filters/AccessTokenAuthenticationFilter.java

```diff
@@ -55,9 +58,13 @@ public void doFilterInternal(@SuppressWarnings("NullableProblems") HttpServletRe
         String token = jwtTool.getTokenFromHttpServletRequest(request);
         if (token != null) {
             try {
+                Optional<User> user = userService.findNotDeactivatedByEmail(jwtTool.getEmailOutOfAccessToken(token));
+                Collection<? extends GrantedAuthority> authorities = null;
+                if (user.isPresent()) {
```

Comment:
Please move this if inside `convertRoles` method.

Reply:
Did you mean that I have to do it the way I did? Since Google says that passing an object enclosed in an Optional into a method is considered bad practice.